.github/workflows/run_main_benchmarks.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-cd $BRANCH_NAME/
+cd "$BRANCH_NAME/" || exit
```

Comment:
For all these cases of `|| exit` I'm concerned that if the command ever fails (which I don't expect) it will be very difficult to diagnose.  Does shellcheck still complain about the missing `|| exit` if you do `#!/bin/bash eux` at the beginning of the script?

Reply:
Using -eux is probably a better way to do it for our case. Updated accordingly